packages/formation-react/src/components/Telephone/Telephone.unit.spec.jsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import Telephone, { CONTACTS, PATTERNS } from './Telephone';
+
+describe('Widget <Telephone />', () => {
+  // custom numbers
+  it('should render a number', () => {
+    const wrapper = shallow(<Telephone contact="8005551212" />);
+    const props = wrapper.props();
+    expect(wrapper.type()).to.equal('a');
+    expect(props.href).to.equal('tel:+18005551212');
+    expect(props['aria-label']).to.equal('800. 5 5 5. 1 2 1 2.');
+    expect(wrapper.text()).to.equal('800-555-1212');
+    wrapper.unmount();
+  });
+  it('should render a number with a leading "1"', () => {
+    const wrapper = shallow(<Telephone contact="1-800-555-1000" />);
+    const props = wrapper.props();
+    expect(props.href).to.equal('tel:+18005551000');
+    expect(props['aria-label']).to.equal('800. 5 5 5. 1000.');
+    expect(wrapper.text()).to.equal('800-555-1000');
+    wrapper.unmount();
+  });
+  it('should throw an error when number does not match pattern', () => {
+    expect(() => {
+      const wrapper = shallow(<Telephone />);
+      wrapper.unmount();
+    }).to.throw('Contact number "" does not match the pattern (###-###-####)');
+  });
+  it('should throw an error when number is less than 10-digits', () => {
+    expect(() => {
+      const wrapper = shallow(<Telephone contact={4321} />);
+      wrapper.unmount();
+    }).to.throw(
+      `Contact number "4321" does not match the pattern (###-###-####)`,
+    );
+  });
+  it('should throw an error when number is more than 10-digits', () => {
+    expect(() => {
+      const wrapper = shallow(<Telephone contact="01234567891" />);
+      wrapper.unmount();
+    }).to.throw(
+      'Contact number "01234567891" does not match the pattern (###-###-####)',
+    );
+  });
+
+  // known numbers
+  it('should render a known number', () => {
+    const wrapper = shallow(<Telephone contact={CONTACTS.DS_LOGON} />);
+    const props = wrapper.props();
+    expect(props.href).to.equal('tel:+18005389552');
+    expect(props['aria-label']).to.equal('800. 5 3 8. 9 5 5 2.');
+    expect(wrapper.text()).to.equal('800-538-9552');
+    wrapper.unmount();
+  });
+  it('should render VA311 (a known number)', () => {
+    const wrapper = shallow(<Telephone contact={CONTACTS.VA_311} />);
+    const props = wrapper.props();
+    expect(props.href).to.equal('tel:+18446982311');
+    expect(props['aria-label']).to.equal('8 4 4. 6 9 8. 2 3 1 1.');
+    expect(wrapper.text()).to.equal('844-698-2311');
+    wrapper.unmount();
+  });
+  it('should render 911 (a known number)', () => {
+    const wrapper = shallow(<Telephone contact={CONTACTS['911']} />);
+    const props = wrapper.props();
+    expect(props.href).to.equal('tel:911');
```

Comment:
Do we know this will call `911` correctly? It's not the easiest one to test. The accepted answer in this StackOverflow [HTML tel: link to emergency number](https://stackoverflow.com/a/61033331) suggests the href should have form `<a href="tel:+1-911">911</a>` as rendered. 

I don't mean to hold up progress--if this has been tested or verified independently, please disregard.

Reply:
I don't know what the correct solution is for `911`. Should it include the `+1`?

Reply:
**If** we link 911, in the US, it should be coded as `<a href="tel:+1-911">911</a>` — as Trevor indicated. 

I'm curious if we provide links to 911 on VA.gov. If we do, what is the use case? My instinct tells me we should not be providing links to call 911 on VA.gov. There are legal considerations that give me pause. Would love to hear more about the use case, though.

Reply:
@jenstrickland @Mottie The VA online scheduling tool does have a 911 link:

![Screen Shot 2020-05-20 at 3 25 36 PM](https://user-images.githubusercontent.com/634932/82488615-30af4000-9aae-11ea-8eb2-1158f5f4ee17.png)

This is a page in our flow for creating a new appointment. I believe the rationale was to try to redirect anyone seeking emergency care, since creating a request online typically has some lag time. @rtwell might have more background.

Reply:
I was fortunate and spoke with a PM that has experience and calmed my worries. Link the 911 for sure. :)